kahuna/public/js/services/api/leases-helper.js

```diff
@@ -0,0 +1,33 @@
+import moment from 'moment';
+
+function readLeases(image) {
+  return image.data.leases.data;
+}
+
+export function getApiImageAndApiLeasesIfUpdated(image, apiImage) {
+    const apiLeases = readLeases(apiImage);
+    const leases = readLeases(image);
+    const apiImageAndApiLeases = {image: apiImage, leases: apiLeases};
+    const isNewlyCreated = leases.lastModified === null;
+    if (isNewlyCreated) {
```

Comment:
No pattern matching/guards in JS :( One day ...

Reply:
would be usefull to move that to TypeScript one day